src/plugins/home/public/application/components/welcome.tsx

```diff
@@ -140,20 +144,48 @@ export class Welcome extends React.Component<Props> {
     }
   };
 
+  private renderBrandingEnabledOrDisabledLogo = () => {
+    if (!this.props.branding.smallLogoUrl) {
+      return (
+        <span className="homWelcome__logo">
+          <EuiIcon type={OpenSearchMarkCentered} size="original" />
+        </span>
+      );
+    } else {
+      return (
+        <div className="homWelcome__customLogoContainer">
+          <img
+            className="homWelcome__customLogo"
+            data-test-subj="welcomeCustomLogo"
+            data-test-image-url={this.props.branding.smallLogoUrl}
+            alt="logo"
```

Comment:
@kgcreative Hi Kevin, for example, what is a good alt here for better accessibility? I am thinking about {branding.title + " logo on welcome page"} so it reads as "opensearch dashboard logo on welcome page", but is that too much information for voice over? (The voice over function on mac speaks super fast)

Reply:
`{branding.title + " logo"}` is enough.

Reply:
okay, thanks!